evergreen.yml

```diff
@@ -270,7 +270,6 @@ functions:
         working_dir: mongosql-odbc-driver
         script: |
           ${prepare_shell}
-          echo "env $(env)"
```

Comment:
This could potentially leak secrets, I never should have had that there permanently, it was for debugging!